int32 video_socket
int32 control_socket
int32 sensor_socket
int32 latency_socket

